commands/dbaas/mongo/cluster/delete.go

```diff
@@ -64,17 +65,17 @@ ionosctl db m c d --all --name <name>`,
 			chosenCluster, _, err := client.Must().MongoClient.ClustersApi.ClustersFindById(context.Background(), clusterId).Execute()
 			if err != nil {
 				wrapped := fmt.Errorf("failed trying to find cluster by id: %w", err)
-				keepGoing := confirm.Ask(fmt.Sprintf("%s, try deleting %s anyways", wrapped.Error(), clusterId))
+				keepGoing := confirm.FAsk(c.Command.Command.InOrStdin(), fmt.Sprintf("%s, try deleting %s anyways", wrapped.Error(), clusterId))
 				if !keepGoing {
 					return wrapped
 				}
 			}
 
-			ok := confirm.Ask(confirmStringForCluster(chosenCluster), viper.GetBool(core.GetFlagName(c.NS, constants.ArgForce)))
+			ok := confirm.FAsk(c.Command.Command.InOrStdin(), confirmStringForCluster(chosenCluster), viper.GetBool(constants.ArgForce))
 			if !ok {
 				return fmt.Errorf("user denied confirmation")
```

Comment:
For some resources we throw an error if user denies confirmation, yet for others we don't.

Let's make a decision and update them all to be the same